Dockerfile-amd64

```diff
@@ -1,4 +1,4 @@
-FROM cgr.dev/chainguard/wolfi-base:latest
+FROM cgr.dev/chainguard/python:latest-dev
```

Comment:
Building off of `wolfi-base` gave me issues related to `pycocotools`. We could probably do this off of `wolfi-base` by adding some of the `apk` packages listed [here](https://edu.chainguard.dev/chainguard/chainguard-images/reference/python/), but for this image I think `python:latest` works fine. We could likely also slim down the image size doing something like the following from the docs. Did confirm the `-dev` version didn't have any CVEs.

```
FROM cgr.dev/chainguard/python:latest-dev as builder

WORKDIR /app

COPY requirements.txt .

RUN pip install -r requirements.txt --user

FROM cgr.dev/chainguard/python:latest
```